CMSgov/onemac
# OY2 22841: Evaluate and Pull Fix for AVScan failure into OneMAC

Story: https://qmacbis.atlassian.net/browse/OY2-22841
Endpoint: See github-actions bot comment

### Details
The MC Review team discovered an issue with the AV Scan code we are all using.  We have implemented the same fix as they did, cleaned up other parts of the code that were not being used, and verified that it correctly discovers if an uploaded file contains a virus.

### Changes
- updated the error handling to go from exception to return value
- Fixed the virus scanning so that it actually works (?!) there were runtime errors, permission errors, and pathing errors
- cleaned up areas where unused code was interfering with readability and my tracking down the issues

### Test Plan
1. Login as a submitting user
2. Navigate to package view 
3. submit a package with attachments
4. Login to AWS console
5. Verify s3 objects for the files uploaded have the virusScan tag and timestamp
6. Navigate to submission view 
7. submit a form with attachments
8. Login to AWS console
9. Verify s3 objects for the files uploaded have the virusScan tag and timestamp
10. Bonus: use the [eicar virus scan test file](https://www.eicar.org/download-anti-malware-testfile/) and verify that the file is correctly tagged as INFECTED.
